cluster-operator/src/main/java/io/strimzi/operator/cluster/model/cruisecontrol/Capacity.java

```diff
@@ -178,8 +228,120 @@ public static Double getSizeInMiB(String size) {
      * @param throughput The String representation of the throughput.
      * @return The equivalent number of kibibytes.
      */
-    public static Double getThroughputInKiB(String throughput) {
+    public static String getThroughputInKiB(String throughput) {
         String size = throughput.substring(0, throughput.indexOf("B"));
-        return StorageUtils.convertTo(size, "Ki");
+        return String.valueOf(StorageUtils.convertTo(size, "Ki"));
+    }
+
+    public static void main(String[] args) {
+        System.out.println(getThroughputInKiB("10000KiB/s"));
+        System.out.println(removeKibSuffix("10000KiB/s"));
+
+    }
```

Comment:
what's this `main` here?

Reply:
Whoops I meant to remove that,  just removed it now